### PR TITLE
Furnace suite, Polyglot, HealthEstimate

### PR DESCRIPTION
**Pull request by mclemente**

### CODE REVIEW

**Review by anthonyronda — APPROVED** [review]
Looks good to me

edit: actually I found problems, sorry!

**Review by ardittristan — CHANGES_REQUESTED** [review]
the module name line, manifest line and closing bracket line start with spaces instead of tabs
Should be tabs

File: package-status.json [hunk 1]
```diff
@@ -29,6 +29,48 @@
 			"deprecated": false,
 			"maintainers": ["sneat"]
 		},
+    "furnace": {
+      "manifest": "module.json",
```
Comment (anthonyronda) [inline package-status.json]: needs to be the installable manifest link

File: package-status.json [hunk 2]
```diff
@@ -29,6 +29,48 @@
...
+			"repo": "https://github.com/League-of-Foundry-Developers/fvtt-module-furnace",
+			"maintained": false,
+			"deprecated": "0.7.10",
+			"maintainers": []
+    },
+    "advanced-macros": {
+      "manifest": "module.json",
```
Comment (anthonyronda) [inline package-status.json]: needs to be the installable manifest link

File: package-status.json [hunk 3]
```diff
@@ -29,6 +29,48 @@
...
+			"repo": "https://github.com/League-of-Foundry-Developers/fvtt-advanced-macros",
+			"maintained": false,
+			"deprecated": false,
+			"maintainers": []
+    },
+    "initiative-double-click": {
+      "manifest": "module.json",
```
Comment (anthonyronda) [inline package-status.json]: needs to be the installable manifest link

File: package-status.json [hunk 4]
```diff
@@ -29,6 +29,48 @@
...
+			"repo": "https://github.com/League-of-Foundry-Developers/fvtt-initiative-double-click",
+			"maintained": false,
+			"deprecated": false,
+			"maintainers": []
+    },
+    "split-journal": {
+      "manifest": "module.json",
```
Comment (anthonyronda) [inline package-status.json]: etc

**Review by anthonyronda — CHANGES_REQUESTED** [review]

**Comment by mclemente** [comment]
I don't even know how I got spaces there, sorry about that, I think it's all fixed now.

**Pull request merged**